CanvasColorSpaceProposal.md

```diff
@@ -202,21 +206,91 @@ The ``getImageData`` method is responsible for converting the data from the canv
 var colorSpace = window.matchMedia("(color-gamut: p3)").matches ? "display-p3" : "srgb";
 </pre>
 
-## Adoption
-Lack of color management and color interoperability is a longstanding complaint about the canvas API.
-Authors of games and imaging apps are expected to be enthusiastic adopters.
+## Resolved Issues
+
+### Additional color spaces, high bit depth, and high dynamic range
+
+Should we support color spaces besides `'srgb'` and `'display-p3'`.
+
+In another proposal.
+This proposal limits its scope to the capabilites of 8-bit buffers.
+Additional color spaces such as ``'rec2020'`` or any HDR color spaces will need higher bit depth support.
+We defer supporting color spaces that require more than 8 bit of precision to a separate proposal that also covers high bit depth and high dyanmic range.
+
+### Custom color profile support
+
+Should we support custom color spaces based on ICC profiles?
+
+Not now.
+Rendering to an arbitrary ICC profile adds a significant level of complexity to the implementation.
+This proposal does not add this capability, but does not close the door to its addition in a future proposal.
+The ``CanvasColorSpaceEnum`` could be change to a ``DOMString`` that could refer any CSS color profile, including CSS Color Module Level 4's [custom profiles](https://www.w3.org/TR/css-color-4/#at-profile).
+
+### Separating chromaticities and transfer functions
+
+Should there be API-level support for mixing chromaticities and transfer functions, including use of no-op transfer functions?
+
+No.
+The color space names are selected to coincide with the CSS Color Module Level 4 [predefined color space names](https://www.w3.org/TR/css-color-4/#predefined).
+Any new syntax for specifying color spaces should be made in that context.
+
+### Handling unrecognized color spaces
+
+Should context creation throw on an unrecognized, non-undefined creation attribute?
+
+No.
+Consider the following example.
+An application calls ``canvas.getContext('2d', {colorSpace:'myNonExistentColorSpace'})``.
+
+* If run on a browser that does not support canvas color spaces the ``colorSpace`` key is ignored, and an sRGB canvas is created.
+* If context creation were to throw on an unrecognized attribute, then this code would fail only on browsers that support canvas color spaces.
+
+Succeeding on a browser that does not support the feature and failing on a browser that does support the feature is an undesirable behavior.
+
+### Naming of color gamuts
+
+The [Media Query APIs](https://www.w3.org/TR/mediaqueries-4/) use the name "p3", while the [CSS Color Module Level 4](https://www.w3.org/TR/css-color-4/#predefined) uses the name "display-p3". Could this divergence be confusing?
+
+Yes and no.
+Indeed it can be confusing that a gamut name and color space name similar but different.
```

Comment:
name similar -> name are similar

Reply:
changed to "are similar but not the same".